training/src/pages/Trainee/TraineeDetail.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import Link from '@material-ui/core/Link';
+import * as moment from 'moment';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import trainee from './data/trainee';
+import { NotFound } from '../../components/NotFound';
+
+const styles = () => ({
+  card: {
+    display: 'flex',
+    margin: '2rem',
+  },
+  found: {
+    display: 'flex',
+    flexDirection: 'row',
+    paddingBottom: '14px',
+  },
+  content: {
+    flex: '1 0 auto',
+  },
+  thumbnail: {
+    padding: '4rem',
+    fontSize: '14px',
+    color: 'white',
+    backgroundColor: 'grey',
+  },
+  button: {
+    display: 'block',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+});
+
+const getDateFormatted = date => moment(date).format('dddd, MMMM Do YYYY, h:mm:ss a');
+
+function Traineefound(props) {
+  const { classes, match } = props;
+  // console.log('----params---', match.params.id);
+  const found = trainee.find((element) => {
```

Comment:
Use find properly, this is not the right way, find will only return element if a function returns true in it.